tests/html/customevent-detail.html

```diff
@@ -1,31 +0,0 @@
-<!doctype html>
```

Comment:
Removed as it was testing the polyfill for CustomEvent, which is not part of html-imports anymore